kmapper/cover.py

```diff
@@ -94,3 +93,104 @@ def find_entries(self, data, cube, verbose=0):
         hypercube = data[np.invert(np.any(entries == False, axis=1))]
 
         return hypercube
+
+
+class CoverBounds(Cover):
+    """
+    Class that allows cover to be defined using upper and lower bounds for given dimensions.
+    """
+
+    def __init__(self,
+                 n_cubes=10,
+                 perc_overlap=0.2,
+                 # Deprecated parameters:
+                 nr_cubes=None,
```

Comment:
rm these parameters, nr_cubes and overlap_perc. only necessary in the base class for backwards compatibility.